profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -227,6 +229,25 @@ function _nexteuropa_varnish_merge_paths_from_purge_rules($purge_rules, $node) {
  * Purges certain paths on the web frontend cache.
  */
 function _nexteuropa_varnish_purge_paths($paths) {
+  try {
+    $settings = _nexteuropa_varnish_get_varnish_settings();
+  }
+  catch (InvalidArgumentException $iae) {
+    $end_sentence = t('Please ask your support team to check the server configuration.');
```

Comment:
We could create a function with this part of code because it's used on two functions (we need to add a parameter for the beginning of the message).
(_nexteuropa_varnish_purge_paths & _nexteuropa_varnish_base_request_headers)